db/post_migrate/20240612073116_swap_field_type_with_field_type_enum_on_user_fields.rb

```diff
@@ -1,14 +1,14 @@
 # frozen_string_literal: true
 
 class SwapFieldTypeWithFieldTypeEnumOnUserFields < ActiveRecord::Migration[7.0]
-  DROPPED_COLUMNS ||= { user_fields: %i[field_type] }
+  # DROPPED_COLUMNS ||= { user_fields: %i[field_type] }
```

Comment:
Maybe add a quick comment on why this was commented? Maybe we can delete it altogether? Or will that mess with the `schema_migrations` table?

Reply:
> Or will that mess with the schema_migrations table?

This was the reason I didn't want to delete it. I'm not sure, but I know deleting normal migrations can cause issues, so I imagine it's the same. I can definitely add some comments both here and in the new migration as well. 👍

Reply:
Okay cool, thanks!